bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -280,70 +270,67 @@ <h2 class="mzp-c-billboard-title">{{ _('Support a healthy internet.') }}</h2>
-            image_url='home/2018/cards/common-voice.png',
-            include_highres_image=True,
+            tag_label=_('Privacy'),
+            title=_('Seven apps that put privacy first'),
+            ga_title='Seven apps that put privacy first',
+            desc=_('Here are seven apps and services that are committed to your privacy and security, including search engines, email providers and more.'),
+            image_url='home/2018/cards/privacy-apps.png',
             aspect_ratio='mzp-has-aspect-16-9',
-            link_url='https://blog.mozilla.org/blog/2018/06/07/parlez-vous-deutsch-rhagor-o-leisiau-i-common-voice/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card'
+            link_url='https://blog.mozilla.org/internetcitizen/2018/08/07/privacy-apps/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card'
         )}}
-
+<!-- 11 -->
         {{ card_block(
-            tag_label=_('Spotlight'),
-            title=_('Working for good: Accel Lifestyles'),
-            ga_title='Working for good: Accel Lifestyles',
-            desc=_('Troubled by fast fashion production processes, Accel’s CEO set out to make change for the better.'),
-            image_url='home/2018/cards/accel-lifestyles.jpg',
+            tag_label=_('Rapid Fire'),
+            title=_('Marshall Erwin talks trust and security'),
+            ga_title='Marshall Erwin talks trust and security',
+            desc=_('Get Marshall’s take on why security, privacy and trust matter so much at Mozilla and Firefox.'),
+            image_url='home/2018/cards/marshall-erwin.jpg',
             include_highres_image=False,
             aspect_ratio='mzp-has-aspect-3-2',
-            link_url='https://blog.mozilla.org/firefox/working-for-good-accel-lifestyle/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card'
+            link_url='https://blog.mozilla.org/internetcitizen/2018/08/09/marshall-erwin-trust/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card'
         )}}
-
+<!-- 12 -->
         {{ card_block(
-            tag_label=_('Internet health'),
-            title=_('7 ways online rights have changed '),
-            ga_title='7 ways online rights have changed ',
-            desc=_('As 2018 hits the halfway point, here is our list of online issues we feel are most important.'),
-            image_url='home/2018/cards/online-rights.png',
-            include_highres_image=True,
+            tag_label=_('Partners'),
+            title=_('WeTransfer keeps community at heart'),
+            ga_title='WeTransfer keeps community at heart',
+            desc=_('WeTransfer, the independent file transfer and sharing service, makes file-sharing easy in Firefox while keeping user privacy at heart.'),
+            image_url='home/2018/cards/we-transfer.png',
```

Comment:
Some image such as this one are pretty huge. Can we resize them more apropriately according to the size of to save on page weight? Resizing images can be a tedious task, but it's probably worthwhile still 😩 

The previous home page cards can probably act as general guidelines for sizing (we still need better docs for this in Protocol 😢)